README-wsl2-windows.md

```diff
@@ -1,118 +1,44 @@
-This method avoids [path length issues]([Error: Too long output directory · Issue #252 · esp-rs/esp-idf-sys (github.com)](https://github.com/esp-rs/esp-idf-sys/issues/252)) and provides a smoother development experience.
-
-## System Requirements
-
-- Windows 10 version 2004 and higher (Build 19041 and above) or Windows 11
-- 64-bit processor with Second Level Address Translation (SLAT)
-- 4GB system RAM or more (8GB recommended)
-- BIOS-level hardware virtualization support must be enabled in the BIOS settings
-- At least 5GB of free disk space for WSL2 and Ubuntu installation
-
-## Basic prerequisite
-
-- Administrative access to your Windows machine
-- Reliable internet connection for downloading necessary components
-
-## Setup Steps
-
-1. Enable WSL2 on Windows
-   - Open "Turn Windows features on or off"
-   - Check "Virtual Machine Platform" and "Windows Subsystem for Linux"
-   - Click OK and restart your computer when prompted
-
-2. Install Ubuntu from Microsoft Store
-   - Open Microsoft Store and search for "Ubuntu"
-	   - As of September 2024, it's based on the Ubuntu 22.04 LTS distribution
-   - Click "Install" to download and install Ubuntu
-
-3. Launch Ubuntu and complete initial setup
-   - Set up a username and password when prompted
-
-4. Update and upgrade Ubuntu packages
-   ```bash
-   sudo apt update && sudo apt upgrade -y
-   ```
-
-5. Install required packages
-   ```bash
-   sudo apt install -y git curl gcc cmake python3 python3-pip pkg-config clang wget flex bison gperf python3-venv ninja-build ccache libffi-dev dfu-util libusb-1.0-0 libssl-dev libusb-1.0-0 libudev-dev
-   ```
-
-6. Install Rust and required tools
-   ```bash
-   curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
-   source $HOME/.cargo/env
-   cargo install cargo-generate 
-   cargo install ldproxy
-   cargo install espup 
-   cargo install espflash 
-   cargo install cargo-espflash (optional)
-   ```
-
-7. Install ESP-IDF and toolchain
-   ```bash
-   espup install
-   source ~/export-esp.sh
-   ```
-
-8. Configure USB access in WSL2
-   - Install [usbipd-win](https://github.com/dorssel/usbipd-win/releases) on Windows
-   - In your WSL2 terminal, run:
+Using WSL2 does not exhibit [long path length issues](https://github.com/esp-rs/esp-idf-sys/issues/252); furthermore, using WSL2 reduces the waiting time between command line cargo invocations and Rust Analyzer operating on the same projects
+
+## Setup
+1. Follow [WSL2 setup guide](https://learn.microsoft.com/en-us/windows/wsl/install) and [wsl2 development enviroment setup guide](https://learn.microsoft.com/en-gb/windows/wsl/setup/environment#file-storage)
+2. Install a linux distro as per the guide or Ubuntu App; - setup and update the packages
+   - Follow [Rust on ESP](https://docs.esp-rs.org/book/) & [ESP-IDF Toolchain setup guide](https://docs.esp-rs.org/book/) for toolchain setup
```

Comment:
Two nits: 
- Can you make this `- Follow...` dash item 3., as it is not a sub-item of the "2. Install a linux distro"
- Can you provide a link to the main README of `esp-idf-template` (with a local path e.g. `../README.md`) rather than to the Book? The Book always tends to lag behind, so it is better to refer to the template itself

Reply:
Appendix in the main README shall be better and a hyperlink at a relevant section can be given.